pkg/registry/apis/preferences/utils/teams.go

```diff
@@ -0,0 +1,12 @@
+package utils
+
+import (
+	"context"
+
+	authlib "github.com/grafana/authlib/types"
+)
+
```

Comment:
Creating an interface that should eventually be implemented by calling an IAM service (currently it makes a direct SQL query)